src/main/java/teammates/ui/controller/InstructorFeedbackResponseCommentEditAction.java

```diff
@@ -97,8 +104,14 @@ protected ActionResult execute() throws EntityDoesNotExistException {
                            + "comment text: " + feedbackResponseComment.commentText.getValue();
         }
 
-        data.comment = feedbackResponseComment;
-
+        // createdAt, lastEditedAt, commentGiverName and lastEditorName are required to generate editedCommentDetails
+        data.comment = updatedComment;
+        data.question = logic.getFeedbackQuestion(response.feedbackQuestionId);
+        data.showCommentToString = StringHelper.toString(updatedComment.showCommentTo, ",");
+        data.showGiverNameToString = StringHelper.toString(updatedComment.showGiverNameTo, ",");
+        data.instructorEmailNameTable = instructorEmailNameTable;
```

Comment:
My idea was to implement everything in `createEditedCommentDetails` without having to call `FeedbackResponseCommentRow` at all, since that is a UI element that we don't need (as we don't use it in the AJAX result). Here:
```java
    public String createEditedCommentDetails(String giverName, String editorName) {
        boolean isGiverAnonymous = Const.DISPLAYED_NAME_FOR_ANONYMOUS_COMMENT_PARTICIPANT.equals(giverName);
        return "From: " + giverName + " [" + TimeHelper.formatDateTimeForSessions(comment.createdAt, sessionTimeZone) + "] "
                + "(last edited "
                + (isGiverAnonymous ? "" : "by " + editorName + " ")
                + "at " + TimeHelper.formatDateTimeForSessions(comment.lastEditedAt, sessionTimeZone) + ")";
    }
```

This way, you only need to do this:
```java
        // createdAt and lastEditedAt are required to generate editedCommentDetails
        data.comment = updatedComment;
        data.sessionTimeZone = session.getTimeZone();
        data.editedCommentDetails = data.createEditedCommentDetails(commentCreatorName, commentEditorName);
```
No need to create an instructor email name table or set any of the other fields - that is a very convoluted way of passing information and should not be resorted to just to reuse a method.